apps/frontend/static_src/scss/main.scss

```diff
@@ -4,6 +4,7 @@
 @import "./variables";
 @import "./functions";
 @import "./mixins";
+@import "./utilities";
```

Comment:
I’ll move this to be at the end of the file, we want utilities to be able to trump other code.